### ISSUE TITLE
Ledger Live Staking Issue

### ISSUE THREAD

**Issue by 555-FILK** [Zondax/ledger-polkadot#74]
Stash account is a Ledger account. Controller account is injected from the Chrome extension to Polkadot.js. Everything works fine with staking. Then when Ledger Live support for Polkadot was released today with staking, I wanted to "move it over" to Ledger Live. I added the Polkadot account via Ledger Live and voila, the stash account appeared but the controller account did not because the private key is not on my Ledger -- makes sense.

Because of this, Ledger Live does not let you manage your stash account because it is controlled by a non-Ledger controller. But, you can view the stash account on Ledger Live and see the transactions, etc. It all seems to be working fine if you want to let it ride. But if you want to make any changes, it doesn't seem like I can do it from Ledger Live.

Attached is a screen grab of the message displayed in the Nominations portion of the account page on Ledger Live. Instead of giving access to nomination features, it displays this message.
![2](https://user-images.githubusercontent.com/80064814/110013570-a2234c80-7cd6-11eb-8119-e3640dd72b6b.jpg)

I am wondering what my next step should be. Create a new controller on Ledger Live and control the stasher that way? Unbond and start again on Ledger Live? Thank you!

**Comment by bvris** [comment]
I have the same issue. I have injected on Polkadot.JS two accounts VIA LEDGER. Stash account and controller account. Now I can import to Ledger Live only injected via ledger Stash account but I do not see and now able to add injected via Ledger controller account in Ledger Live. It could be some issue with Polkadot.JS or some issue with account type/index when setting up on Polkadot.JS first time via Ledger add feature. I've tried today adding via Ledger new account on Polkadot.JS and it show as ledger instead of injected which is strange, as nothings changed. Used same add via ledger feature.

**Comment by bvris** [comment]
> Stash account is a Ledger account. Controller account is injected from the Chrome extension to Polkadot.js. Everything works fine with staking. Then when Ledger Live support for Polkadot was released today with staking, I wanted to "move it over" to Ledger Live. I added the Polkadot account via Ledger Live and voila, the stash account appeared but the controller account did not because the private key is not on my Ledger -- makes sense.
> 
> Because of this, Ledger Live does not let you manage your stash account because it is controlled by a non-Ledger controller. But, you can view the stash account on Ledger Live and see the transactions, etc. It all seems to be working fine if you want to let it ride. But if you want to make any changes, it doesn't seem like I can do it from Ledger Live.
> 
> Attached is a screen grab of the message displayed in the Nominations portion of the account page on Ledger Live. Instead of giving access to nomination features, it displays this message.
> ![2](https://user-images.githubusercontent.com/80064814/110013570-a2234c80-7cd6-11eb-8119-e3640dd72b6b.jpg)
> 
> I am wondering what my next step should be. Create a new controller on Ledger Live and control the stasher that way? Unbond and start again on Ledger Live? Thank you!

hey, just try what I did. Via Polkadot.JS change the controller using your ledger injected/added accounts back to stash account. That's all. Now you can open up ledger live an continue managing everything on ledger live :)

**Comment by 555-FILK** [comment]
I created a new Ledger address on polkadot.js and switched the controller from the injected extension address to the new Ledger address. So now, my stash account is a Ledger address and my controller account is a Ledger address.

Then, on Ledger Live, I deleted my Polkadot stash account and re-added my stash account. The issue persists, but now the address that Ledger Live shows as the address controlling my stash account lists the new Ledger address that I created.
